src/frontend/src/lib/components/networks/NetworksSwitcher.svelte

```diff
@@ -31,7 +32,17 @@
 	aria-label={$i18n.networks.title}
 >
 	<div class="w-full h-full md:w-[28px] md:h-[28px]">
-		{#if nonNullish($selectedNetwork?.icon)}
+		{#if isNullish($selectedNetwork)}
```

Comment:
What about inlining instead of duplicating properties?

```
<Img
				src={$selectedNetwork?.icon ?? chainFusion}
				alt={replacePlaceholders($i18n.core.alt.logo, {
					$name: $selectedNetwork?.name ?? $i18n.networks.chain_fusion
				})}
				width="100%"
				height="100%"
				rounded
			/>
```

 WDYT?

Reply:
yes! i have avoided it to keep the IF logic, but I prefer it too